pyne/variancereduction.py

```diff
@@ -3,9 +3,13 @@
 """
 import numpy as np
 from pyne.particle import mcnp
-from mcnp import Wwinp
+from .mcnp import Wwinp
 from pyne.mesh import Mesh, MeshError, HAVE_PYMOAB
-from itertools import izip
+# default zip in python3 behaves as itertools.izip as python2
+try:
+    from itertools import izip as zip
+except:
```

Comment:
Let's except an `ImportError` specifically here

Reply:
Will do in the next commit.

Reply:
But in Python3 this will raise an exception, right?

Reply:
Yes, for python3 there will be an exception for import `izip`. But when that happens, it means we do have `zip` we want, then we dont't need to do anything with this exception.

Reply:
Agreed. I think that handling the exception with `pass` is fine, but that we should only do that for the specific exception `ImportError`. 


```Python
try:
    from itertools import izip as zip
except ImportError:
    pass
```

Reply:
Got it. Will make this change in the next update.